.github/release.yml

```diff
@@ -2,6 +2,9 @@ changelog:
   exclude:
     labels:
       - ignore-changelog
+      - type/test
+      - type/chore
```

Comment:
below it was `type/chores` while here it is missing the `s`

Reply:
nice catch, thanks